installation/Developer-guide.md

```diff
@@ -92,7 +92,7 @@ Use `exit` to exit from the bash terminal.
 If you have made any changes to docker backend files and just want to restart the docker backend, instead of stopping and restarting the complete environment. Open a new terminal and run this script
 
 ```
-docker-compose -f docker-compose.yml -f docker-compose-dev.yml -f restart backend
+docker-compose -f docker-compose.yml -f docker-compose.dev.yml -f restart backend
```

Comment:
```suggestion
docker-compose -f docker-compose.yml -f docker-compose.dev.yml restart backend
```
I think this was another issue with the suggested command. 🙂

Reply:
yes, thank you for pointing it out.